docs/core/v3.0.rst

```diff
@@ -1036,13 +1036,16 @@ following mandatory names:
     extension and dereferences to a human-readable representation of
     the specification.  The ``type`` is required and its value is
     defined by the extension. The ``fallback`` is optional
-    and, if provided, its value must be one of the data type
-    identifiers defined in this specification. If an implementation
-    does not recognise the extension, but a ``fallback`` is present,
-    then the implementation may proceed using the ``fallback`` value
-    as the data type. For fallback types that do not correspond to base
-    known types, extensions can fallback on a raw number of bytes using
-    the raw type (``r*``).
+    and, if provided, its value must be one or a list of the data type
+    identifiers defined in this specification or an extension. If
+    an implementation does not recognise the extension or specific data
+    type, but a ``fallback`` is present, then the implementation may proceed
+    using the first known ``fallback`` value as the data type. If no fallback
+    types are given or they do not correspond to base known types, extensions
+    can fallback on a raw number of bytes using the raw type (``r*``).
```

Comment:
This says "if no fallback types are given" but then says that a raw type may be used.  Instead, it can simply suggest that for fixed-size data types, if there is no more specific fallback available, then a raw type may be specified as a fallback.

It should also note that the default list of fallbacks is defined by the data type extension, but may be overridden by the user.